fuel-tx/src/contract.rs

```diff
@@ -32,6 +32,12 @@ fn next_multiple<const N: usize>(x: usize) -> usize {
 pub struct Contract(#[derivative(Debug(format_with = "fmt_truncated_hex::<16>"))] Vec<u8>);
 
 impl Contract {
+    /// The `ContractId` of the contract with empty bytecode, zero salt, and empty state root.
+    pub const EMPTY_CONTRACT_ID: ContractId = ContractId::new([
```

Comment:
It would be nice if this was all calculated in a const block instead of being hardcoded. Is this already possible or are the fn's needed to generate this ID not easily usable in const? If so, we could consider using `lazy_static!` as well.

Reply:
It is possible to constantly get `salt`, `root`, and `state_root`, but the hash function itself is not const. 

I added a unit test to verify the empty contract id.